templates/base/head_script.tmpl

```diff
@@ -47,3 +47,12 @@ If you introduce mistakes in it, Gitea JavaScript code wouldn't run correctly.
 	{{/* in case some pages don't render the pageData, we make sure it is an object to prevent null access */}}
 	window.config.pageData = window.config.pageData || {};
 </script>
+
+<script type="module">
+	const browserSyncJson = JSON.stringify({lc: window.navigator.language, tz: new Date().getTimezoneOffset()});
+	if (window.config.pageData.browserSyncJson != null && window.config.pageData.browserSyncJson !== browserSyncJson) {
+		document.cookie = '_gitea_bsj=' + encodeURIComponent(browserSyncJson) + '; path=/'; // tell the backend about locale and timezone
+		document.body.style.setProperty('display', 'none', 'important'); // hide the page to prevent flash during reload
+		setTimeout(() => window.location.reload(), 1000); // use a setTimeout to simulate a slow network
```

Comment:
I would not do this reload, just live with the fact that it works only on second page load.

Reply:
Yup, I could also agree so.

Reply:
ps: cookie won't be a problem, the CSRF token and Gitea session cookie are all there, unless only the `document.cookie` doesn't work ....